components/brave_new_tab_ui/components/default/binance/index.tsx

```diff
@@ -0,0 +1,254 @@
+    const {
+      initialAsset,
+      initialFiat,
+      initialAmount,
+      fiatShowing,
+      currenciesShowing
+    } = this.state
+    const isUS = userTLD === 'us'
+    const currencies = isUS ? this.usCurrencies : this.comCurrencies
+
+    return (
+      <>
+        <Copy>
+          {getLocale('binanceWidgetBuyCrypto')}
+        </Copy>
+        <BuyPromptWrapper>
+          <FiatInputWrapper>
+            <FiatInputField
+              type={'text'}
+              placeholder={getLocale('binanceWidgetBuyDefault')}
+              value={initialAmount}
+              onChange={this.setInitialAmount}
+            />
+            <FiatDropdown
+              disabled={isUS}
+              itemsShowing={fiatShowing}
+              onClick={this.handleFiatChange}
+            >
+              <FiatDropdownLabel>
+                {initialFiat}
+              </FiatDropdownLabel>
+              <CaratDropdown hide={isUS}>
+                <CaratDownIcon />
+              </CaratDropdown>
+            </FiatDropdown>
+            {
+              fiatShowing
+              ? <AssetItems isFiat={true}>
+                  {this.fiatList.map((fiat: string, i: number) => {
+                    if (fiat === initialFiat) {
+                      return null
+                    }
+
+                    return (
+                      <AssetItem
+                        key={`choice-${fiat}`}
+                        isLast={i === (currencies.length - 1)}
+                        onClick={this.setInitialFiat.bind(this, fiat)}
+                      >
+                        {fiat}
+                      </AssetItem>
+                    )
+                  })}
+                </AssetItems>
+              : null
+            }
+          </FiatInputWrapper>
+          <AssetDropdown
+            itemsShowing={currenciesShowing}
+            onClick={this.toggleCurrenciesShowing}
+          >
+            <AssetDropdownLabel>
+              {initialAsset}
+            </AssetDropdownLabel>
+            <CaratDropdown>
+              <CaratDownIcon />
+            </CaratDropdown>
+          </AssetDropdown>
+          {
+            currenciesShowing
+            ? <AssetItems>
+                {currencies.map((asset: string, i: number) => {
+                  if (asset === initialAsset) {
+                    return null
+                  }
+
+                  return (
+                    <AssetItem
+                      key={`choice-${asset}`}
+                      isLast={i === (currencies.length - 1)}
+                      onClick={this.setInitialAsset.bind(this, asset)}
+                    >
+                      {asset}
+                    </AssetItem>
+                  )
+                })}
+              </AssetItems>
+            : null
+          }
+        </BuyPromptWrapper>
+        <ActionsWrapper>
+          <ConnectButton onClick={onBuyCrypto.bind(this, initialAsset, initialAmount, initialFiat)}>
```

Comment:
If this component's goal is to open a new tab, then it should be an anchor instead of a button

Reply:
changed to `<a>`